### PR TITLE
parse partonomy

### PR DESCRIPTION
**Pull request by mccalluc**
No change to indexing right now... just a demo of how I plan to use the partonomy, before this code gets entangled with everything else.

@shirey :
- Do you feel comfortable with including the partonomy this way? ... or would it be better not to write to a gitignored directory? Could just check it in, or possibly use a submodule, but this seems simpler and clearer.

@bherr2 :
- Are you comfortable with how I'm parsing the data, and the structural assumptions I'm making? ie, one parentless root node, assume there is only one label and parent on each node, etc.

Towards: https://github.com/hubmapconsortium/portal-ui/issues/1572

### CODE REVIEW

**Review by bherr2 — APPROVED** [review]
Makes sense and looks good!

**Comment by mccalluc** [comment]
Removing Bill has reviewer: @yuanzhou as codeowner should be sufficient.

File: src/elasticsearch/addl_index_transformations/portal/demo_partonomy.py
Comment (bherr2) [inline src/elasticsearch/addl_index_transformations/portal/demo_partonomy.py:14]: Switch URL to https://cdn.jsdelivr.net/gh/hubmapconsortium/hubmap-ontology@1/ccf-partonomy.jsonld so that it gets the latest version (a new version may appear at end of March but definitely end of May).

**Review by bherr2 — CHANGES_REQUESTED** [review]

File: src/elasticsearch/addl_index_transformations/portal/demo_partonomy.py
Comment (mccalluc) [reply src/elasticsearch/addl_index_transformations/portal/demo_partonomy.py:14]: @bherr2 : I don't think that's the behavior we want: I really prefer to pin dependencies and have the behavior of code be consistent until we make an explicit change.

If this doesn't seem correct to you, let's talk on the call this afternoon.

Comment (bherr2) [reply src/elasticsearch/addl_index_transformations/portal/demo_partonomy.py:14]: Your call. Not how I'd do it, but I understand your reasoning.

**Review by bherr2 — APPROVED** [review]

**Comment by mccalluc** [comment]
Branch extended and new PR filed: #242. Closing.

**Pull request closed without merging**
